regolith/helpers/l_currentappointmentshelper.py

```diff
@@ -85,7 +85,7 @@ def sout(self):
                         print("No grant found for {}".format(grantid))
                     try:
                         accountnr = grant.get("account", grant["alias"])
-                    except:
+                    except NotImplemented:
```

Comment:
This will be a NotImplemented error.  For now maybe just put `Exception` though we would actually like to know how it fails.